src/components/TaskEdit.tsx

```diff
@@ -92,6 +89,8 @@ class TaskEdit extends React.PureComponent<PropsType> {
       this.state.location = event.location ? event.location : '';
       this.state.description = event.description ? event.description : '';
       this.state.timezone = event.timezone;
+      console.log(event);
+      this.state.tags = event.categories ? event.categories.join(' ') : '';
```

Comment:
I know it's probably way too early too comment on this, so feel free to ignore, but I'd rather not have free text it, but rather a sort of a way to select existing tags (multiselect popup?). Maybe in addition we can also have free text (comma delimited though, not space), but I feel like it would need a nice autocomplete, so probably easier to just defer that to a later stage.